server/api/accountRouter.ts

```diff
@@ -13,4 +14,36 @@ router.get("/", async (req: Request, res: Response, next: NextFunction) => {
   }
 });
 
+router.post("/", async (req: Request, res: Response, next: NextFunction): Promise<void>=> {
+  try {
+    const {accountType, accountName, institution, balance} : AccountAttributes = req.body
+    // const createAccount = 
+    await Account.create({
+      accountType,
+      accountName,
+      institution, 
+      balance
+    })
+    res.sendStatus(204)
+  } catch (err) {
+    res.sendStatus(404);
+    next(err);
+  }
+});
+
+router.delete("/:accountId", async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+  try {
+    const accountId: string = req.params.accountId
+    //soimething weird happens since 
+    //accountId starts string but ends up number??
+    //so TS forces it to be string idk
+    const accountToDelete = await Account.findByPk(accountId)
+    accountToDelete?.destroy()
```

Comment:
This will send it ahead even if it is not destroyed raid? With the ?. it passes that portion of the try/catch even if it fails I think.